### ISSUE TITLE
setup_log is not setup correctly due to disable_existing_loggers

### ISSUE THREAD

**Issue by mpkocher** [PacificBiosciences/pbcommand#37]

**Comment by pb-cdunn** [comment]
I think the problem is partly one of convention.

``` python
log = logging.getLogger(__name__)
...
def main(argv=sys.argv):
    log.info("Starting {f} version {v} pbcommand example dev app".format(f=__file__, v=__version__))
    p = get_contract_parser()
    return pbparser_runner(argv[1:],
                           p,
                           _args_runner, # argparse runner func
                           _resolved_tool_contract_runner, # tool contract runner func
                           log, # log instance
                           setup_log # setup log func
                           )
```

`setup_log()` will attach a handler, but that logger is already non-root. Using the root logger solves that.

But the larger problem is how to get debug logging.

Personally, I am a fan of `logging.ini` files, using [`logging.config.fileConfig()`](https://docs.python.org/2/library/logging.config.html) (which can take a fileObj instead of a filename, even though that's not clear from the docs).

**Comment by pb-cdunn** [comment]
Addressed by #39.

(When you merge, you can add "resolves #37" or "fixes #37" or "closes #37 in the merge-message to close the issue automatically.)

**Comment by mpkocher** [comment]
Thanks. Good to know.
